web/src/pages/apd/proposed-budget/IncentivePayments.test.js

```diff
@@ -14,107 +15,120 @@ import {
   setIncentiveEPPayment
 } from '../../../redux/actions/editApd';
 
-describe('incentive payments component', () => {
-  const props = {
+const defaultProps = {
+  data: {
```

Comment:
you don't really need data and totals because you are using the connected component now